client/landing/jetpack-cloud/components/site-offset/README.md

```diff
@@ -0,0 +1,75 @@
+# loaded applySiteOffset
+
+This module provides support for using applySiteOffset inside React components with including the necessary selectors and query components.
```

Comment:
I'm not entirely sure but I think we can remove the `"with"` word from `"...React components with including..."`.

Reply:
It is supposed to be "without" 🤦 "without including the necessary selectors and query components." A small typo that completely changes the meaning.